tests/helpers/perf_run_helper.rb

```diff
@@ -816,6 +816,10 @@ def baseline_assert(atop_result, gatling_result)
     # global atop results and per process atop results
     baseline = get_baseline_result
     baseline&.each do |key, value|
+      # orchestration has a jump in memory usage in 2019.2 due to plan executor
```

Comment:
Do we have a reference to this in docs, code, or jira tickets that can be included here?

Reply:
done